charts/aws-fsx-csi-driver/templates/node-serviceaccount.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.node.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.node.serviceAccount.name }}
+  labels:
+    {{- include "aws-fsx-csi-driver.labels" . | nindent 4 }}
+  {{- with .Values.node.serviceAccount.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+{{- end }}
+---
```

Comment:
Does the service account for the node not need any RBAC permissions?

Reply:
no, but it will in the future if we implement instance metadata workaround where the node pod gets the instance id and such by `Getting` its `Node` versus trying to query instance metadata on local node.